spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -62,7 +62,6 @@ Provides CDI annotations:
 |@Compensate
 |Indicates that the method should be invoked if the LRA is cancelled.
 |Optional
-
```

Comment:
I think the empty line is fine here. Asciidoctor normally separates the table rows with the empty line in the source code.